src/confdb/converter/ConverterFactory.java

```diff
@@ -8,7 +8,7 @@
 	private String classHeader = "Ascii";
 	private boolean untrackedFlag = false;
 	private String additionalParameter4ConfigurationWriter = null;
-
+	//DUMY COMMENT
```

Comment:
I'm curious... why this ?

Reply:
I added it so I could check some problems that we had trying to compare branches before this.  Just to see weather commit reflects on the branch in comparison. It should be removed.